README.md

```diff
@@ -67,6 +67,44 @@ To create a custom FastIR Artifacts collector (custom artifact definitions and c
 - add more options to the `fastir_artifacts.ini` file for instance `library = True` and  `exclude = BrowserCache,WindowsSearchDatabase`
 - zip the `fastir_artifacts` folder and ship it
 
+## Custom Artifact Types
+
+FastIR Artifacts supports the following artifact types in addition to the types defined by the [Digital Forensics Artifact Repository](https://github.com/ForensicArtifacts/artifacts).
+
+### FileInfo
+
+The FileInfo artifact type can be used to collect metadata about files instead of collecting the files themselves:
+
+```yaml
+name: System32 Metadata
+doc: Metadata about dll and exe files in System32.
+sources:
+- type: FILE_INFO
+  attributes:
+    paths:
+    - '%%environ_systemroot%%\System32\*.dll'
+    - '%%environ_systemroot%%\System32\*.exe'
+    - '%%environ_systemroot%%\System32\**\*.dll'
+    - '%%environ_systemroot%%\System32\**\*.exe'
+    separator: '\'
+supported_os: [Windows]
+```
+
+It collects the following information (stored in a JSONL file using [Elastic Common Schema](https://www.elastic.co/guide/en/ecs/current/index.html)):
+
+* MD5 hash
+* SHA-1 hash
+* SHA-256 hash
+* Mime type
+* File size
+* Imphash (PE only)
+* Compilation Date (PE only)
```

Comment:
Codacy found an issue: [[unordered-list-marker-style] Marker style should be `-`](https://app.codacy.com/manual/muller.gael/fastir_artifacts/pullRequest?prid=5498175)